offlinevalidator/offlinevalidator_test.go

```diff
@@ -336,8 +337,18 @@ func TestInvalidMapProperty(t *testing.T) {
 	assert.False(t, validateResources(resources, &spec, &sink, deadProp, deadRes), "This resource should be valid")
 }
 
-func createResourceWithNestedProperties(resourceType string, propertyName string, nestedPropertyValue map[string]interface{}) template.Resource {
+func TestFindingAllowedValues(t *testing.T) {
+	sink = logger.Logger{}
+	data := make(map[string]interface{})
+
+	data["AllowedValues"] = ""
+	data["Type"] = "AWS::EC2::VPC::Id"
+	parameters := createParameters("Incorrect", data)
+
+	assert.True(t, findingAllowedValues(parameters, &sink), "This template should be invalid")
```

Comment:
`findingAllowedValues ` tells nothing about template validity, it  only checks if template has allowed values. Please change this message.